spec.html

```diff
@@ -147,7 +147,7 @@ <h1>
         1. <ins>Let _previousContextMapping_ be AsyncContextSwap(_cell_.[[AsyncContextSnapshot]]).</ins>
         1. <ins>Let _result_ be Completion(HostCallJobCallback(_callback_, *undefined*, « _cell_.[[HeldValue]] »)).</ins>
         1. <ins>AsyncContextSwap(_previousContextMapping_).</ins>
-        1. <ins>Return _result_.</ins>
+        1. <ins>? _result_.</ins>
```

Comment:
```suggestion
        1. <ins>Perform ? _result_.</ins>
```
(rather than just a bare `?`) otherwise, `ReturnIfAbrupt(_result_)`.